src/protocolsupport/protocol/typeremapper/tile/TileEntitySkullRemapper.java

```diff
@@ -3,19 +3,26 @@
 import java.util.List;
 import java.util.function.Consumer;
 
+import org.bukkit.Bukkit;
 import org.bukkit.Material;
 import org.bukkit.block.data.BlockData;
 import org.bukkit.block.data.Rotatable;
 
+import protocolsupport.ProtocolSupport;
 import protocolsupport.api.MaterialAPI;
+import protocolsupport.api.ProtocolSupportAPI;
+import protocolsupport.api.ProtocolType;
+import protocolsupport.api.ProtocolVersion;
 import protocolsupport.protocol.typeremapper.legacy.LegacyBlockFace;
```

Comment:
It looks like you are adding a bunch of unnecessary imports. Try using an editor such as IntelliJ that can help you point out unused imports.

Thanks for keeping up the spirit and working with us to improve the patch!